owasp-security-logging-log4j/pom.xml

```diff
@@ -42,6 +42,11 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
```

Comment:
What is the need for this spring-test dependency? I'm leery of introducing a new dependency especially as this library is intended to be used by non-spring projects as well.

Reply:
All tests do pass with this dependency removed. 

@friedeas can this dependency be eliminated? If you want a test harness for Spring Boot, that is something that could be implemented as a separate module.

Reply:
log4j-core-test in the version 2.20.0 has a dependency to spring-test. I saw it was resolved to version 5.3.24, as this version is known to have serious security vulnerabilities, I set the version to 5.3.29. I could remove the dependency specification but the result would be less secure I think.

Reply:
@augustd Do you agree to better use a updated version of the transitive dependencies of log4j-core-test (spring-test etc.), or should I remove the version update?

Reply:
As this is a transitive dependency introduced through log4j-core-test, the correct way to resolve the issue is by upgrading the dependency in the [log4j-core-test project](https://github.com/apache/logging-log4j2/blob/2.x/log4j-core-test/pom.xml). 

If we want to set an explicit version of spring-test in this project then it needs to have `scope = test` so it does not get included in the released artifact.